app/app.js

```diff
@@ -0,0 +1,43 @@
+// app.js
+
+
+const express = require('express');
+
+
+ /* Importing the controller files */
+ const {getTopics} = require('../controllers/topics.controller');
+
+
+ /* Middleware functions */
+const app = express();
+app.use(express.json())
```

Comment:
You don't need this until dealing with POST/PATCH requests. Please delete it until then